src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3575,6 +3575,52 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
         return nullptr;
     }
 
+    ASR::asr_t* handle_intrinsic_str(Allocator &al, Vec<ASR::call_arg_t> args,
+                                        const Location &loc) {
+        ASR::expr_t *arg = nullptr;
+        ASR::expr_t *res_value = nullptr;
+        ASR::ttype_t *arg_type = nullptr;
+        if (args.size() > 0) {
+            arg = args[0].m_value;
+            arg_type = ASRUtils::expr_type(arg);
+        }
+        ASR::ttype_t *res_type = ASRUtils::TYPE(ASR::make_Character_t(al, loc,
+                                    1, 1, nullptr, nullptr , 0));
+        if (!arg) {
+            throw SemanticError("it must be one argument", loc);
```

Comment:
```suggestion
            return ASR::make_StringConstant_t(al, loc, s2c(al, ""), res_type);
```

Reply:
yes, i saw it in tests and changed already.